src/clj/rems/auth/fake_login.clj

```diff
@@ -1,23 +1,38 @@
 (ns rems.auth.fake-login
-  (:require [clojure.string :as str]
-            [compojure.core :refer [GET defroutes]]
+  (:require [compojure.core :refer [GET defroutes]]
             [hiccup.util :refer [url]]
             [rems.layout :as layout]
-            [rems.db.core :as db]
-            [rems.db.users :as users]
+            [rems.auth.oidc :as oidc]
+            [rems.config :refer [env]]
+            [rems.db.test-data-users :refer [+fake-user-data+ +demo-user-data+]]
+            [rems.common.util :refer [assoc-some-in]]
+            [rems.ga4gh :as ga4gh]
             [ring.util.response :refer [redirect]]))
 
+(defn get-fake-login-users []
+  (condp = (:fake-authentication-data env)
+    :test +fake-user-data+
+    :demo +demo-user-data+))
```

Comment:
this function allows easy users substitution from outside if needed (see tests for example in `test/clj/rems/api/test_user_settings.clj`)